core-blocks/freeform/editor.scss

```diff
@@ -1,4 +1,4 @@
-.wp-block-freeform.editor-rich-text__tinymce {
+.wp-block-freeform.blocks-rich-text__tinymce {
```

Comment:
Good catch @mcsf should we instead rename this one `core-blocks-rich-text__tinymce` to follow our style conventions? similar to #6684

Reply:
Works for me!

Reply:
Amended.